lib/types/index.js

```diff
@@ -518,10 +521,14 @@ export type Profile = {
 }
 
 export type OtpServer = {
-  internalUrl: string,
+  id: string,
+  instanceType: ?string,
+  internalUrl: ?string,
   name: string,
+  projectId: ?string,
   publicUrl: string,
-  s3Bucket: ?string
+  s3Bucket: ?string,
+  targetGroupArn: ?string
```

Comment:
This is missing `instanceCount` field that is present in the datatools-server PR.